public/app/features/explore/TraceView/components/DetailsPanel/AccordianReferences.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { css, cx } from '@emotion/css';
+import * as React from 'react';
+
+import { Field, LinkModel } from '@grafana/data';
+import { Icon, useStyles2 } from '@grafana/ui';
+
+import AccordianKeyValues from '../TraceTimelineViewer/SpanDetail/AccordianKeyValues';
+import { TraceSpanReference } from '../types/trace';
+import { ubMb1 } from '../uberUtilityStyles';
+import ReferenceLink from '../url/ReferenceLink';
+
+const getStyles = () => {
+  return {
+    AccordianKeyValues: css`
+      margin-left: 10px;
+    `,
+    AccordianReferences: css`
+      label: AccordianReferences;
+      position: relative;
+      margin-bottom: 0.25rem;
+    `,
+    AccordianReferencesContent: css`
+      label: AccordianReferencesContent;
+      padding: 0.5rem 0.5rem 0.25rem 0.5rem;
+    `,
+    itemContent: css`
+      padding: 0.25rem 0.5rem;
+      display: flex;
+      width: 100%;
+      justify-content: space-between;
+    `,
+    item: css`
+      &:nth-child(2n) {
+        background: #f5f5f5;
+      }
+    `,
+    debugInfo: css`
+      letter-spacing: 0.25px;
+      margin: 0.5em 0 0;
+      flex-wrap: wrap;
+      display: flex;
+      justify-content: flex-end;
+    `,
+    debugLabel: css`
+      margin: 0 5px 0 5px;
+      &::before {
+        color: #bbb;
+        content: attr(data-label);
+      }
+    `,
+    serviceName: css`
+      margin-right: 8px;
+    `,
+    title: css`
+      display: flex;
+      align-items: center;
+      gap: 4px;
+    `,
+  };
+};
+
+export type AccordianReferencesProps = {
+  data: TraceSpanReference[];
+  highContrast?: boolean;
+  interactive?: boolean;
+  openedItems?: Set<TraceSpanReference>;
+  onItemToggle?: (reference: TraceSpanReference) => void;
+  createFocusSpanLink: (traceId: string, spanId: string) => LinkModel<Field>;
+};
+
+type ReferenceItemProps = {
+  data: TraceSpanReference[];
+  interactive?: boolean;
+  openedItems?: Set<TraceSpanReference>;
+  onItemToggle?: (reference: TraceSpanReference) => void;
+  createFocusSpanLink: (traceId: string, spanId: string) => LinkModel<Field>;
+};
+
+// export for test
+export function References(props: ReferenceItemProps) {
```

Comment:
Again this is mostly a copy of original `References` component with design tweaks to suit card layout with tabs.